subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -172,7 +172,7 @@ Options include:
  * <<working_with_files#sec:creating_archives_example,Creating custom archive tasks>>
  * Using a suitable community plugin from the https://plugins.gradle.org/[Gradle Plugin Portal]
 
-If you're lucky and don't have many plugins or much in the way of customisation in your Maven build, you can simply run
+If you don't have many plugins or much in the way of customisation in your Maven build, you can simply run
```

Comment:
Suggestion:

_If your maven build follows standard conventions and does not apply any specific plugins, you can simply run_

Reply:
Adapted but not 100% like suggested. After all plugins like surefire and others are not an issue at all.